### PR TITLE
Implementing frontend.numpy.isrealobj

### LINKED ISSUE #11171: isrealobj
working on (#1527->  isrealobj)

### PR DESCRIPTION
**Pull request by Sarvesh-Kesharwani**
Fixes #11171

### CODE REVIEW

**Comment by Sarvesh-Kesharwani** [comment]
@aditya-hari Could you please guide me on the update i have done.

thnax

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_logic/test_truth_value_testing.py [hunk 1]
```diff
@@ -141,3 +141,29 @@ def test_numpy_isfortran(
...
+):
+    input_dtype, x = dtype_and_x
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        frontend=frontend,
+        test_flags=test_flags,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        a=x[0],
+    )
```
Comment (aditya-hari) [inline ivy_tests/test_ivy/test_frontends/test_numpy/test_logic/test_truth_value_testing.py:169]: Add a newline after this

**Comment by aditya-hari** [comment]
Hey! Thanks for the contribution. The test currently only generates float as the dtype. You should use something with complex, given that this is supposed to check for complex types

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_logic/test_truth_value_testing.py (hunk 1 above)
Comment (Sarvesh-Kesharwani) [reply ivy_tests/test_ivy/test_frontends/test_numpy/test_logic/test_truth_value_testing.py:169]: sure

**Comment by Sarvesh-Kesharwani** [comment]
Hi, @aditya-hari 
could you pls review the update.

thanks

**Comment by aditya-hari** [comment]
The tests are failing with ```TypeError: isrealobj() got an unexpected keyword argument 'a'```

**Comment by aditya-hari** [comment]
Additionally, please go through the contribution guideline to appropriately format this PR by linked this to an issue, which is subsequently linked to the ToDo issue

**Comment by Sarvesh-Kesharwani** [comment]
> 

Hey, @aditya-hari i went through the contribution video to find out way to link a PR with it's issue, but the video had explanation only for members, so i looked at other contributor's closed PRs, and i have linked the issue to this PR by mentioning issue's code above
chrome_s8pcYCfKhh

please confirm if it is the correct way

thank you

**Comment by aditya-hari** [comment]
LGTM!

**Pull request merged**